docs/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 – `New` – *Menu Config* – Default and HTML items now support hints
 – `Fix` — Deleting whitespaces at the start/end of the block
 – `Improvement` — *Types* — `BlockToolConstructorOptions` type improved, `block` and `config` are not optional anymore
+- `Improvement` — Link creating on mobile devices
```

Comment:
```suggestion
- `Improvement` — Creating links on Android devices: now the caret will be positioned after the selected fragment when the Enter key is pressed
```

Reply:
@neSpecc Are you sure about this readme entry?

As far as I understand, we are just adding a [hint to the virtual keyboard](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/enterkeyhint) on how to handle the "Next" button. By default, this is moving the input to the next field of the form, but with my change we simply ask you to approve the input.

Without this change, it will not be the `Enter` key, but the key of the `Next Field` - you can see this in my videos.
![Screenshot 2024-07-01 at 00 20 34](https://github.com/codex-team/editor.js/assets/1257284/7c182094-e92a-4840-9506-8e550f7a7872) ![Screenshot 2024-07-01 at 00 20 45](https://github.com/codex-team/editor.js/assets/1257284/145cc96f-1ea5-4ce5-9c42-a664d2c9f786)

We don't move the cursor, we don't do anything - we just explain what we want from the virtual keyboard with this hint. And we can't be sure how the client device will handle this.

Reply:
The changelog should explain end-user experience, which is about the new caret behavior, as I saw on your video. Maybe replace "Enter is pressed" with "Done is pressed"?

Reply:
@neSpecc You are right!

How about this line?

> Add type hint `done` on creating links to make it work correctly on mobile devices.

Reply:
It's a kind of technical detail, but 'correctly' is not explained

Reply:
@neSpecc New version:

> Add type hint `done` for the link input to prevent switching to the next field on mobile devices.

Maybe you just enter any description you like? I just want to have this change in the next version =)

Reply:
```
Creating links on Android devices: now the mobile keyboard will have an "Enter" key for accepting the inserted link.
```

Reply:
@neSpecc Done!